test/language_list_test.exs

```diff
@@ -0,0 +1,61 @@
+defmodule LanguageListTest do
+  use ExUnit.Case
+  doctest LanguageList
+
```

Comment:
Discover [`setup_all/1`](https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html#setup_all/1)!
```elixir
setup_all do
  λs =
    LanguageList.new()
    |> LanguageList.add("Clojure")
    |> LanguageList.add("Haskell")
  [langs: λs]
end
```

and then in tests:
```elixir
test "removes the first element", %{langs: langs}  do
  assert ["Closure"] = LanguageList.remove(langs)
end
```